migrations.json

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
have you added it or got it added automatically?

Reply:
Ah, this is generated by nx update migration. But could also be added to gitignore. But also does no harm when added